app/models/form526_submission.rb

```diff
@@ -490,8 +491,8 @@ def last_remediation
   private
 
   def conditionally_submit_form_4142
-    if Flipper.enabled?(:disability_compensation_production_tester, User.find(user_uuid))
-      Rails.logger.info("submit_form_4142 call skipped for submission #{submission.id}")
+    if Flipper.enabled?(:disability_compensation_production_tester, OpenStruct.new({ flipper_id: user_uuid }))
+      Rails.logger.info("submit_form_4142 call skipped for submission #{id}")
```

Comment:
is `#{id}` supposed to be `#{submission.id}`?

Reply:
haha NOPE! That was our original mistake, and it was throwing an exception that prevented the logging entirely. We're already in the context of the submission (just as `user_uuid` is here).

Reply:
Ah-ha! Sounds good. 👍🏻